timings/timings.go

```diff
@@ -0,0 +1,106 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package timings
+
+import (
+	"fmt"
+	"time"
+)
+
+// Subject is a type of the activity being measured; available subjects are defined below.
+type Subject string
```

Comment:
I'm slightly wary of `Subject` - how will that be used in the future (e.g. will it grow more categories?). I guess I mostly wodner what value does it add to the data. As it is now (with just the two subjects) it feels to me like not much but maybe I just need some examples to see the value :)

Reply:
@pedronis yeah, I'm not sure either. FWTW that's the "what is it about" aspect we discussed, and that it describes entire tree.

Reply:
tags should be enough for that